activiti-engine/src/main/java/org/activiti/engine/impl/cmd/integration/RetrieveIntegrationContextCmd.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.engine.impl.cmd.integration;
+
+import org.activiti.engine.impl.interceptor.Command;
+import org.activiti.engine.impl.interceptor.CommandContext;
+import org.activiti.engine.impl.persistence.entity.integration.IntegrationContextEntity;
+
+public class RetrieveIntegrationContextCmd implements Command<IntegrationContextEntity> {
+
+    private String correlationId;
```

Comment:
@erdemedeiros ok let's remove the correlation ID and use the UUID provided by the executionId, that will be secure enough.

Reply:
Ok. But should we send the `executionId` as `executionId` or should we hide this behind another name? 
- exposing it as `executionId` would simplify things for people who are already familiar with the concept
- exposing it with a different name gives the flexibility to rely on other information later.
In any case, the `executionId` is already exposed through the `RuntimeService`, so it's not really an internal concept and we can probably expose it in the message.

Reply:
@erdemedeiros let's use the UUID executionId